evm-ext/src/native_impl.rs

```diff
@@ -107,7 +113,19 @@ fn evm_transact_inner(
     // TODO: does it make sense for EVM gas to be 1:1 with MoveVM gas?
     let gas_limit: u64 = context.gas_balance().into();
 
+    // Struct external to the EVM to capture transfer events.
+    // This is used for bookkeeping token balances between Move and EVM.
+    //
+    // It needs to use a `RefCell` for interior mutability because the
+    // EVM handler hooks require the closures to be immutable.
+    // The usage of interior mutability is safe because execution of the single
+    // EVM instance is single threaded.
```

Comment:
This would probably make a good docstring on the `EthTransfersLogger` struct.

Reply:
Moved in https://github.com/MovedNetwork/op-move/pull/319/commits/4b1845a594e661cbbc0db45dc6ce5dd181271738